### ISSUE TITLE
External cost doesn't work after upgrade from 1.52 to 1.53.2 release

### ISSUE THREAD

**Issue by gmlexx** [kubecost/cost-analyzer-helm-chart#315]
**Describe the bug**
External cost doesn't work after upgrade from 1.52 to 1.53.2 release

**To Reproduce**
1. Open Allocations menu
2. See zero external costs

**Expected behavior**
External costs data, based on resources tags

**Screenshots**
N/A

**Collect logs (please complete the following information):**
cost-model container log:
```
E0304 10:40:32.146692       1 main.go:488] cloud expenses: error: cloud expense: error reading Athena columns: athena query failed: {
  Query: "SHOW COLUMNS IN mycosts",
  QueryExecutionContext: {
    Database: "athenacurcfn_my_costs"
  },
  QueryExecutionId: "fc97a4d1-6733-43f4-af20-194d4338e9ac",
  ResultConfiguration: {
    OutputLocation: "s3://aws-athena-query-results-kubecost/fc97a4d1-6733-43f4-af20-194d4338e9ac.txt"
  },
  StatementType: "UTILITY",
  Statistics: {
    QueryQueueTimeInMillis: 50,
    TotalExecutionTimeInMillis: 50
  },
  Status: {
    State: "QUEUED",
    SubmissionDateTime: 2020-03-04 10:40:32 +0000 UTC
  },
  WorkGroup: "primary"
}
```

**Comment by dwbrown2** [comment]
Thanks, @gmlexx! Also being tracked here: https://github.com/kubecost/cost-model/issues/371

**Comment by dwbrown2** [comment]
Fixed in v1.53.3. Let me know if we can get you anything else here.
